frontend/src/app/TelemetryService.ts

```diff
@@ -47,7 +54,7 @@ const createTelemetryService = () => {
 export const ai = createTelemetryService();
 export const getAppInsights = () => appInsights;
 
-export function filterStaticFiles(envelope: ITelemetryItem) {
+export function isBlacklistedFile(envelope: ITelemetryItem) {
```

Comment:
nit: if we want to use a more inclusive naming scheme, we can call this `isBlockedFile` or something like that

Reply:
Ty for flagging this Elisa!